cmd/config/get.go

```diff
@@ -3,16 +3,19 @@ package config
 import (
 	"context"
 	"fmt"
+	_nethttp "net/http"
```

Comment:
I know you didn't write this but I have no idea why this is renamed `_nethttp` 🤷 

There might have been a good reason at some point but I don't see it now. If you wouldn't mind cleaning this up, that would be great. Always good to leave things a little cleaner than you found them.